components/squad/squad_row.lua

```diff
@@ -104,6 +122,8 @@ function SquadRow:newteam(args)
 		if mw.ext.TeamTemplate.teamexists(newteam) then
 			cell:wikitext(mw.ext.TeamTemplate.team(args.newteam:lower(),
 				args.newteamdate or ReferenceCleaner.clean(args.leavedate)))
+
+			self.lpdbData['newteam'] = mw.ext.TeamTemplate.teampage(newteam)
 		elseif self.options.useTemplatesForSpecialTeams then
 			if newteam == 'retired' or newteam == 'inactive' then
 				cell:wikitext(Template.safeExpand(mw.getCurrentFrame(), 'Team/' .. newteam))
```

Comment:
should we maybe also store special new teams somehow?

Reply:
Need to decide on that Liquipedia-wide

Reply:
I think that the "special teams" (retired, military) should be seen as roles and not teams.

Reply:
Even in the display? Because that doesn't really make sense. You want to show a leavedate etc

Reply:
What I'm thinking:

Has leavedate set (with reference where applicable) and is in the former section.
NewTeam = <blank>
NewTeamRole = _Retired_

Correct me if I'm wrong, but the special teams are just Italic text?

Reply:
That does make sense but requires a new LPDB property, which has been a hassle lately. I guess we could save it in extradata for now?

Reply:
Imo, the db Field/property newteamrole should be created regardless how we handle special teams.
Extradata is fine for now in my book

Reply:
I've been thinking about this and this is what I think we need in LPDB:

- newteam
- newteamrole
- newstatus

"retired", "inactive", etc should go into `newstatus`

Reply:
okay for me